### PR TITLE
(doc) Fix typo and language in Contributing guidelines

### PR DESCRIPTION
**Pull request by grv87**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ The Chocolatey team has very explicit information here regarding the process for
...
+Chocolatey is a large ecosystem and each component has their own location for submitting issues and enhancement requests. While the website (the community package repository) may be all you know for packages, it represents only a tiny fraction of existing packages (organizations typically maintain and host their own packages internally). This is the repository for choco.exe (the client CLI tool) for Chocolatey, which spans multiple types of environments.
 
-Please follow this decision criteria to see if you are in the right location or if you should head to a different location to submit your request.
+Please follow these decision criteria to see if you are in the right location or if you should head to a different location to submit your request.
```
[the remark this answers is not in the document]
Comment (grv87) [reply CONTRIBUTING.md]: I've checked with [Wiktionary](https://en.wiktionary.org/wiki/criteria), they say the standard is that `criteria` is plural for `criterion`.
Also, there are several items in the list below that text.

**Comment by grv87** [comment]
Done

**Comment by ferventcoder** [comment]
Thanks, this has been merged into master at fc937ob

**Comment by StephNdevo** [comment]
Merged

**Pull request merged**